docs/project/goals.md

```diff
@@ -507,7 +507,7 @@ interfaces with business logic in Carbon, although this direction can tolerate
 slightly more constraints both in supported features and performance overhead.
 In all cases, the particular performance overhead imposed by moving between C++
 and Carbon will need to be easily exposed and understood by developers. While a
-given piece code only needs to be migrated once, we expect interoperability to
+given piece of code only needs to be migrated once, we expect interoperability to
 be invoked continuously to support migrated code and will thus remain important
 for most developers.
```

Comment:
```suggestion
given piece of code only needs to be migrated once, we expect interoperability
to be invoked continuously to support migrated code and will thus remain
important for most developers.
```

Note we use pre-commit / prettier for formatting (https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md). I'll apply this here, but if you plan on contributing regularly we'd appreciate you running these tools in order to simplify review.